src/serializer/functions.js

```diff
@@ -168,6 +168,30 @@ export class Functions {
     applyOptimizedReactComponents(this.realm, this.writeEffects, environmentRecordIdAfterGlobalCode);
   }
 
+  getDeclaringAdditionalFunction(functionValue: ECMAScriptSourceFunctionValue) {
```

Comment:
We decided back in January to rename additional functions to optimized functions, but even now we are still adding new names that incorporate additional. Sigh.